frontend/src/index.js

```diff
@@ -7,9 +7,9 @@ const {Elm} = require('./Main');
 Elm.Main.init({
   flags: {
     elasticsearchMappingSchemaVersion: parseInt(process.env.ELASTICSEARCH_MAPPING_SCHEMA_VERSION),
-    elasticsearchUrl: process.env.ELASTICSEARCH_URL || 'https://nixos-search-5886075189.us-east-1.bonsaisearch.net:443',
-    elasticsearchUsername : process.env.ELASTICSEARCH_USERNAME || 'z3ZFJ6y2mR',
-    elasticsearchPassword : process.env.ELASTICSEARCH_PASSWORD || 'ds8CEvALPf9pui7XG',
+    elasticsearchUrl: process.env.ELASTICSEARCH_URL || 'https://nixos-search-7-1733963800.us-east-1.bonsaisearch.net:443',
+    elasticsearchUsername : process.env.ELASTICSEARCH_USERNAME || 'aWVSALXpZv',
+    elasticsearchPassword : process.env.ELASTICSEARCH_PASSWORD || 'X8gPHnzL52wFEekuxsfQ9cSh',
```

Comment:
Is it safe to have this password in clear text BTW? What privileges does this account have?

Reply:
Safe. These are credentials with read only permission.